mdata/aggmetric.go

```diff
@@ -596,3 +596,11 @@ func (a *AggMetric) GC(chunkMinTs, metricMinTs uint32) bool {
 	}
 	return false
 }
+
+func (a *AggMetric) gcAggregators(chunkMinTs, metricMinTs uint32) bool {
+	ret := true
+	for _, agg := range a.aggregators {
+		ret = agg.GC(chunkMinTs, metricMinTs, a.lastWrite) && ret
```

Comment:
instead of iterating over all aggregators this could just return `false` on the first one that returns `false`

Reply:
Since agg.GC() had time based side effects (e.g. persisting) even if it returns false, it's important to trigger for all.